starter-code/index.html

```diff
@@ -42,44 +38,40 @@ <h1>Pizza Builder</h1>
     <!-- Price -->
     <aside class="panel price">
       <h2>Your pizza's price</h2>
-      <b><!-- Put the base price of the pizza here --> cheese pizza</b>
+      <b>
+        Base price : {{basePrice}} $
+        <br>
+        <!-- Put the base price of the pizza here -->cheese pizza</b>
       <ul>
+        <li v-for="ingredient in ingredients" v-if="ingredient.selected=== true">
+          {{ingredient.name + " " + ingredient.price}}
+        </li>
         <!-- List all selected ingredients with their price. Use a computed property. -->
       </ul>
-      <strong><!-- Display the total price. Use a computed property. --></strong>
+      <strong>
+        {{togglePrice}}
```

Comment:
Cool, but a better name would probably be `totalPrice`, since it's a computed prop that gives you that value, and you're using it declaratively